ERRATA.md

```diff
@@ -24,7 +24,7 @@ This will be addressed in the next version of the blueprint guide.
 
 #### Naming
 - The Service Account naming is not aligned to the blueprint guide. Naming will be modified accordingly in a future release.
-- The infrastructure pipeline project naming (`prj-buN-c-infra-pipeline`) is not aligned to the blueprint guide(`prj-buN-c-sample-infra-pipeline`). Naming will be modified accordingly in a future release.
+- The infrastructure pipeline project naming (`prj-ml-buN-c-infra-pipeline`) is not aligned to the blueprint guide(`prj-ml-buN-c-sample-infra-pipeline`). Naming will be modified accordingly in a future release.
```

Comment:
what is "buN"?

Reply:
I looked for other references like this but I couldn't find them, I think it could be something linked to "Naming" as mentioned in the note, but I'm not sure